doc/source/index.rst

```diff
@@ -9,28 +9,42 @@
 Fast Azimuthal Integration using Python
 =======================================
 
-PyFAI is a python libary for azimuthal integration of X-Ray/neutron scattering data acquired with area-detectors. 
+PyFAI is a python libary for azimuthal integration of X-ray/neutron/electron scattering data acquired with area-detectors. 
 For this, images needs to be re-binned in polar coordinate systems. 
 Additional tools are provided to calibrate the experimental setup, i.e. define where the detector is positionned in space considering the sample and the incident beam. 
 
 .. figure:: img/overview.png
    :align: center
    :alt: PyFAI is about regridding image in polar space.
 
-The sub-title of the project, *the space-folder*, is related to the expertise acquired in re-distributing the signal acquired in one geometry (often cartesian) 
-into another one (often polar) while propagating properly the associated error. Unlike interpolation, the flux is conserved in those transformations, and can be 
-used for any type of space transformation, including image distortion and many more.     
+The core idea is to re-distribute the signal acquired with the experimental geometry 
+into a geometry suitable for further analysis like Rietveld refinement for power data 
+or Inverse Fourier Transform for SAXS data.    
+Unlike interpolation, this re-distribution conserves the signal, its variance and 
+can used for other type of transformation like distortion correction.  
+
+Since, the alignement of the beam, of the sample and of the detector can never be perfect,
+pyFAI tries to cope with it by calibrating their relative position using a 
+reference sample material (called calibrant). 
+After calibration, the geometry can be saved in a *poni-file* and used to perform azimuthal averaging
+of several samples. 
+The geometry used by pyFAi is described in this scheme:
+     
+.. figure:: img/PONI.png
+   :align: center
+   :alt: The geometry used by pyFAI is inspired by SPD
```

Comment:
There's quite a bit of text in this image, which may be important for understanding geometry & PONI files. A user with a screenreader probably won't get any of that, just the alt text. It might be worth either having more detailed alt text, or a caption with a link - "See $SECTION for more on how geometry is represented"

Reply:
I improved the description... but a sketch is worth 1000 words !